assembl/static2/js/app/components/common/tree.jsx

```diff
@@ -319,34 +117,35 @@ class Tree extends React.Component {
         <div style={style}>
           {index > 0 ? <SeparatorComponent /> : null}
           <Child
-            {...childData}
-            contentLocaleMapping={parent.props.contentLocaleMapping}
             key={childData.id}
-            lang={parent.props.lang}
+            {...childData}
+            identifier={identifier}
+            phaseId={phaseId}
+            lang={lang}
             rowIndex={index}
+            contentLocaleMapping={parent.props.contentLocaleMapping}
             InnerComponent={InnerComponent}
             InnerComponentFolded={InnerComponentFolded}
             SeparatorComponent={SeparatorComponent}
             nuggetsManager={this.nuggetsManager}
             listRef={this.listRef}
             cache={this.cache}
-            identifier={identifier}
-            phaseId={phaseId}
           />
         </div>
       </CellMeasurer>
     );
   };
 
   render() {
-    const { contentLocaleMapping, data, lang, noRowsRenderer } = this.props;
+    const { contentLocaleMapping, data, lang, noRowsRenderer, InnerComponentHeight } = this.props;
     return (
       <WindowScroller>
         {({ height, isScrolling, onChildScroll, scrollTop }) => (
           <AutoSizer
             disableHeight
             onResize={() => {
               this.cache.clearAll();
+              // $FlowFixMe listRef is not null
```

Comment:
Higher above you set listRef: List | null = null; Couldn't you check listRef not null before calling the function and then would flow pass ?